linkis-engineconn-plugins/pipeline/src/main/scala/org/apache/linkis/manager/engineplugin/pipeline/executor/PipelineEngineConnExecutor.scala

```diff
@@ -60,7 +60,7 @@ class PipelineEngineConnExecutor(val id: Int) extends ComputationExecutor with L
     var succeedTasks = 1
     val newOptions = new util.HashMap[String, String]()
     newOptions.putAll(EngineConnObject.getEngineCreationContext.getOptions)
-    engineExecutorContext.getProperties.foreach { keyAndValue =>
+    engineExecutorContext.getProperties.asScala.foreach { keyAndValue =>
       newOptions.put(keyAndValue._1, keyAndValue._2.toString)
     }
     newOptions.foreach({ case (k, v) => logger.info(s"key is $k, value is $v") })
```

Comment:
`PipelineEngineConnExecutor.scala:66: value foreach is not a member of java.util.HashMap[String,String]`